addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6129,7 +6129,17 @@ msgctxt "#13036"
 msgid "Failed for {0:s}"
 msgstr ""
 
-#empty strings from id 13037 to 13049
+#: addons/skin.estuary/xml/DialogButtonMenu.xml
+msgctxt "#13037"
+msgid "Toggle Windows HDR On"
+msgstr ""
+
+#: addons/skin.estuary/xml/DialogButtonMenu.xml
+msgctxt "#13038"
+msgid "Toggle Windows HDR Off"
```

Comment:
```suggestion
msgid "Toggle Display HDR Off"
```

Reply:
Display is HDR all the time.... It is Windows that has a switch on the Control Panel to "prevent use"

This control toggles "Windows switch ON/OFF" 
And This control **only** appears in Kodi for Windows platform
Android not has this problem because no exist "Windows switch" in Android

Reply:
please change as suggested. If a feature appears only for one platform doesn't mean that it will not be apear for others in the future.

Reply:
Done